src/object_properties.cpp

```diff
@@ -71,6 +73,9 @@ void ObjectProperties::serialize(std::ostream &os) const
 	writeF1000(os, weight);
 	writeV3F1000(os, collisionbox.MinEdge);
 	writeV3F1000(os, collisionbox.MaxEdge);
+	writeV3F1000(os, selectionbox.MinEdge);
+	writeV3F1000(os, selectionbox.MaxEdge);
+	writeU8(os, pointable);
```

Comment:
Should this be done as part of this PR? I have seen other new entity properties added without the need for that. I'might not too sure how to go about that, TBH.